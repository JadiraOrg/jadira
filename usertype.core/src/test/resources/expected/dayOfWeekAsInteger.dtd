<!ELEMENT dataset (
    DAYOFWEEKASINTEGER*)>

<!ELEMENT DAYOFWEEKASINTEGER EMPTY>
<!ATTLIST DAYOFWEEKASINTEGER
    ID CDATA #REQUIRED
    DAYOFWEEK CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

