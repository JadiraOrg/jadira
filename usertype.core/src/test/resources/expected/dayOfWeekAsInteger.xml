<?xml version='1.0' encoding='UTF-8'?>
<dataset>
  <dayOfWeekAsInteger ID="0" NAME="test_0" DAYOFWEEK="1"/>
  <dayOfWeekAsInteger ID="1" NAME="test_1" DAYOFWEEK="2"/>
  <dayOfWeekAsInteger ID="2" NAME="test_2" DAYOFWEEK="7"/>
</dataset>
